picard/metadata.py

```diff
@@ -135,10 +134,11 @@ def compare_to_release_parts(self, release, weights):
             except ValueError:
                 pass
             else:
-                if "title" in weights:
-                    b = int(release.medium_list[0].medium[0].track_list[0].count)
-                else:
-                    b = int(release.medium_list[0].track_count[0].text)
+                if 'media' in release:
```

Comment:
It might be possible that total tracks was taken from file metadata and while comparing, the compared release does not have a media tag. (I had an error while testing, which is why I add this)